Src/FluentAssertions/Equivalency/Property.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Reflection;
+using FluentAssertions.Common;
+
+namespace FluentAssertions.Equivalency
+{
+    /// <summary>
+    /// Provides an ISelectedMemberInfo for PropertyInfo objects
+    /// </summary>
+    #pragma warning disable CA1716
```

Comment:
IDE0055 would like this to be without leading spaces. (like all other #pragmas in the source code)